File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map = {}

        for i in range(len(nums)):
            lookingFor = target - nums[i]
            if lookingFor in map:
                return [map[lookingFor], i]
            map[nums[i]] = i
        return false
        

File: Data Structures & Algorithms/concatenation-of-array/submission-1.js
class Solution {
    /**
     * @param {number[]} nums 
     */
    getConcatenation(nums) {

        let i = 0
        let n = nums.length
        
        while (i < n) {
            nums.push(nums[i])
            i++
        }
        return nums
    }
}

